internal/credentials/credentials.go

```diff
@@ -186,8 +281,27 @@ func (cs *CredentialsService) save(table CredentialTable) error {
 	return nil
 }
 
-// Loads the CredentialTable
+// Loads the CredentialTable with keyring and env values
 func (cs *CredentialsService) load() (CredentialTable, error) {
+	table, err := cs.loadFromKeyRing()
+	if err != nil {
+		return nil, err
+	}
+
+	// insert a possible environment variable credential before returning
+	record, err := cs.EnvCredentialRecordFactory()
+	if err != nil {
+		return nil, err
+	}
+	if record != nil {
+		table[EnvVarGUID] = *record
```

Comment:
What should we do if there's a name or URL conflict here?

Reply:
I've added error checks for the get, just to make sure the environment variable doesn't overlap with anything in the table.